app/Kconfig

```diff
@@ -103,6 +103,21 @@ config USB_NUMOF_EP_WRITE_RETRIES
 config USB_HID_POLL_INTERVAL_MS
     default 1
 
+config ZMK_USB_BOOT
+    bool "USB Boot Protocol Support"
+    default y
+    select USB_HID_BOOT_PROTOCOL
+    select USB_DEVICE_SOF
+
+if ZMK_USB_BOOT
+
+# Declare boot keyboard interface
+config USB_HID_PROTOCOL_CODE
+    default 1
```

Comment:
menuconfig mentions this Kconfig as deprecated:

```
Name: USB_HID_PROTOCOL_CODE
Prompt: HID Boot Interface protocol code (DEPRECATED)
Type: int
Value:

Help:

  This option is deprecated.
  Please use usb_hid_set_proto_code() instead.
```

so we should probably switch to using that API instead.